.github/workflows/build_docker.yml

```diff
@@ -93,9 +93,30 @@ jobs:
 
     - name: Build BlackDuck Docker Images with dotnet core
       run: |
```

Comment:
I have the feeling that the tag `dotnetcore` should remain, otherwise people pulling this image will start facing issues as removing this tag is a breaking change. @JeroenKnoops, what you think?

Reply:
The default dotnetcore should be 3.1.102 I think, so this change looks fine! Still good to verify! :)

Reply:
The dotnetcore should point to the latest version available. If people want to stick into specific versions, they should use the versioned images.

Reply:
The dotnetcore should point to the latest version available. If people want to stick into specific versions, they should use the versioned images. 

Also I don't think 3.0 image (which has the tag) is used by anyone. Because nuget_inspector of blackduck requires 2.2 runtime which was missing in the 3.0 image.